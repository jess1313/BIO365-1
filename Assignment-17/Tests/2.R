suppressPackageStartupMessages(suppressWarnings(suppressMessages(source("2.R"))))

set.seed(0)

print(getPGreater(rnorm(1000), -6))
print(getPGreater(rnorm(1000), -4))
print(getPGreater(rnorm(1000), -2))
print(getPGreater(rnorm(1000), 0))
print(getPGreater(rnorm(1000), 2))
print(getPGreater(rnorm(1000), 4))
print(getPGreater(rnorm(1000), 6))