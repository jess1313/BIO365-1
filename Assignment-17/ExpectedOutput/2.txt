[1] 1
[1] 0.9999398
[1] 0.9848853
[1] 0.5010223
[1] 0.0248799
[1] 4.610964e-05
[1] 3.336284e-10
